### ISSUE TITLE
Saving an asset drops password

### ISSUE THREAD

**Issue by Wikiphil** [johnnyq/itflow#294]
- I create an asset, includes a login and password then save. Password is saved. (capture 1 and 2)
![1](https://user-images.githubusercontent.com/84292480/149688832-f81dbc1c-6c60-4bb8-99b2-202ddb03109a.png)
![2](https://user-images.githubusercontent.com/84292480/149688834-bf33e1e5-dd8f-4abe-a060-d050ae825ac3.png)

- Click on the key to see the login / password, it's there. (capture 3) and password should be hidden by default! :-)
![3](https://user-images.githubusercontent.com/84292480/149688835-ebfccbbe-4935-4e0a-93d2-782dd960fc2f.png)

- Go back in the asset, touch nothing (capture 4) and save, password is dropped (capture 5) or it's now hidden! 
![4](https://user-images.githubusercontent.com/84292480/149688836-4e2329ce-6bed-4582-b30d-080042fef045.png)
![5](https://user-images.githubusercontent.com/84292480/149688838-f17afeb8-a08c-4d75-99bf-81b28b6382cc.png)

- Confirmed under Logins, password (capture 6, ok, my bad, I used "password" as Password, but there is a Capital P on the empty cell)
![6](https://user-images.githubusercontent.com/84292480/149688839-9ac1afee-35c8-4e1e-abf2-cfe87c2f3488.png)

**Comment by johnnyq** [comment]
good catch this must of happened when we refactored the way passwords are stored and retrieved from the database

**Comment by Wikiphil** [comment]
Sorry bother with this stupid question:

What is the best way with git command to clone back the repo without drop my config?

**Comment by johnnyq** [comment]
go into the directory with ITFlow and do a git pull this will update everything but wont touch your uploads directory or your config.php file
